Lib/test/test_exceptions.py

```diff
@@ -1027,6 +1027,17 @@ def g():
         self.assertIsInstance(v, RecursionError, type(v))
         self.assertIn("maximum recursion depth exceeded", str(v))
 
+
+    @cpython_only
+    def test_crashcan_recursion(self):
+        def foo():
+            o = object()
+            for x in range(1000000):
```

Comment:
1000000 looks arbitrary. Maybe use `sys.getrecursionlimit() * 1000`? At least, could you write `1_000_000` for readability please?

Reply:
> 1000000 looks arbitrary

1000000  is enough to trigger the stack overflow. `sys.getrecursionlimit() * 1000` is not always `1000000` depending on `sys.getrecursionlimit()` so I will leave as it is. I will use `1_000_000` though